### ISSUE TITLE
atom - parse author error

### ISSUE THREAD

**Issue by BiteBit** [rla/fast-feed#12]
Hi, i come again.
I parse http://githubengineering.com/atom.xml. But got the result of author concatenated.
The author's uri, name, email was concatenated
So i read the RFC4278 and RFC5023. 
```
 A Person construct is an element that describes a person,
   corporation, or similar entity (hereafter, 'person').
   atomPersonConstruct =
      atomCommonAttributes,
      (element atom:name { text }
       & element atom:uri { atomUri }?
       & element atom:email { atomEmailAddress }?
       & extensionElement*)
```

**Comment by rla** [comment]
This is result from #11 that now concatenates multiple nodes instead of taking first of them. Before it took `name` node or the text node or whatever came first. I go fix it and add author_uri and author_email.

**Comment by rla** [comment]
Added new Atom author parsing. When `author` contains a `name` node then it uses the `name` node contents and parses also `uri` and `email` nodes. Otherwise it uses the old behavior. Added tests, updated documentation and published the new version 1.2.0.
